src/Sylius/Bundle/CoreBundle/Installer/Setup/LocaleSetup.php

```diff
@@ -89,7 +89,9 @@ private function getLanguageName(string $code): ?string
         $region = null;
 
         if (count(explode('_', $code, 2)) === 2) {
-            [$language, $region] = explode('_', $code, 2);
+            $codeParts = explode('_', $code, 2);
+            $language = $codeParts[0];
+            $region = $codeParts[1] ?? null;
```

Comment:
If you pass `pl` the language is `pl` and the region is `null`.
If you pass `pl_PL` the language is `pl` and the region is `PL`. 

So I guess we can't delete it.

Reply:
I meant the condition itself and what's above leaving only the
```php
$codeParts = explode('_', $code, 2);
$language = $codeParts[0];
$region = $codeParts[1] ?? null;
```
since it covers both cases

Reply:
Ouch, right 🤷🏼‍♂️